src/standalone/Soil/energy_hydrology.jl

```diff
@@ -274,9 +253,26 @@ function ClimaLand.make_compute_imp_tendency(
         @. dY.soil.ϑ_l =
             -(divf2c_rre(-interpc2f(p.soil.K) * gradc2f(p.soil.ψ + z)))
 
+        # Heat equation RHS
+        divf2c_heat = Operators.DivergenceF2C(
+            top = Operators.SetValue(Geometry.WVector.(heat_top_flux_bc)),
+            bottom = Operators.SetValue(Geometry.WVector.(heat_bottom_flux_bc)),
```

Comment:
Just a note though that this will allocate two surface Fields on every Jacobian update. It would generally be better to cache these Fields somewhere, and you could also use the same cached Field for the top and bottom BCs.

Reply:
Thanks, Dennis! good catch!